workflow/commit-messages.md

```diff
@@ -0,0 +1,34 @@
+# Commit Guidelines
+
+As a general guideline you can follow the idea that `a single commit should enclose a single problem/feature that has been fixed/added`.
```

Comment:
Backticks will place `<code>` around the block. Please don't use them for emphasis or quotes. Either use `"` double-quotes or `*` emphasis.